ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -208,6 +229,10 @@ runDataDiffusion tracers
                             ] ++ dnsSubThreads
 
   where
+    traceException :: IO a -> IO a
+    traceException f = catch f $ \(e :: SomeException) -> do
+      traceWith dtDiffusionInitializationTracer (DiffusionErrored e) 
+      throwIO e
```

Comment:
Please leave an empty line between definitions. We even use two empty lines at times, especially when we use a single empty line between different cases of the same definition.  This makes things much easier to read.

Reply:
Done!